pkgs/by-name/cm/cmd-unwrapped/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cmd-wrapped";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "YiNNx";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-9GyeJFU8wLl2kCnrwZ+j+PwCRS17NvzgSCpulhXHYqQ=";
+  };
+
+  cargoSha256 = "sha256-i6LgLvLMDF696Tpn4yVA1XNuaTrABLVg3SgclHBq6Go=";
+
+  meta = with lib; {
+    description = "Find out what the past year looks like in commandline";
+    homepage = "https://github.com/YiNNx/cmd-wrapped";
+    license = licenses.mit;
+    platforms = platforms.linux;
```

Comment:
```suggestion
    platforms = platforms.unix;
```

Reply:
Just a newbie’s doubt, when should I use unix?

Reply:
There are darwin binaries in https://github.com/YiNNx/cmd-wrapped/releases/tag/0.1.1, so certainly not `platforms.linux`. For Rust packages, `meta.platforms` already has a default value, so I'll just use that.